linera-storage/src/db_storage.rs

```diff
@@ -1157,3 +1163,168 @@ where
         Ok(Self::new(database, wasm_runtime, clock))
     }
 }
+
+#[derive(Default, Debug, Eq, PartialEq, Serialize, Deserialize)]
+#[repr(u8)]
+enum SchemaDescription {
+    /// Version 0, all the blobs, certificates, confirmed blocks, events and network description on the same partition.
+    /// This is marked as default since it does not exist in the old scheme and would be obtained from unwrap_or_default
+    #[default]
+    Version0SingleBlobPartition,
+    /// Version 1, spreading by ChainId, CryptoHash, and BlobId.
+    Version1MultiBlobPartition,
+}
+
+/// The key containing the schema of the storage.
+const SCHEMA_ROOT_KEY: &[u8] = &[0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233];
```

Comment:
What are these bytes made of? Please use either:
* a unit test to keep the bytes consistent with the formula.
* `once_cell::sync::Lazy` to compute the bytes from the formula (once) at runtime.